server/app/services/applicant/ApplicantService.java

```diff
@@ -967,6 +1151,7 @@ private ImmutableMap<Path, String> stageNormalUpdates(
               failedUpdatesBuilder.put(currentPath, update.value());
             }
             break;
+          case SERVICE_AREA:
```

Comment:
**question (blocking):** it looks like we're writing the service area check results both here and below on 1179-1184, why is that?

Reply:
This was a mistake leftover from an iteration, thank you for catching it!